### PR TITLE
Haewon - LA112.U

### PR DESCRIPTION
**Pull request by d3y419**

### CODE REVIEW

**Comment by d3y419** [comment]
is this now the correct direction of the pull request?

File: R/zchunk_LA112.U.R [hunk 1]
```diff
@@ -28,37 +28,30 @@ FILE = "energy/A12.U_curves"))
...
+ add_comments("Uranium supply and extraction costs imported from GCAM 3.0 data") %>%
+ add_comments("All global supply is assigned to a single region") %>%
  add_legacy_name("L112.RsrcCurves_Mt_R_U") %>%
- add_precursors("precursor1", "precursor2", "etc") %>%
- # typical flags, but there are others--see `constants.R` 
- add_flags(FLAG_NO_TEST, FLAG_LONG_YEAR_FORM, FLAG_NO_XYEAR) ->
+ add_precursors("common/GCAM_region_names",
+                "energy/A12.U_curves") %>%
+ # typical flags, but there are others--see `constants.R`
+ add_flags(FLAG_NO_XYEAR) ->
```
Comment (bpbond) [inline R/zchunk_LA112.U.R]: Can get of this line entirely--no flags needed at all.

**Comment by bpbond** [comment]
Thanks Sonny. Should we open an issue re updating the 2003 data, to revisit later? If so please do so.

**Comment by bpbond** [comment]
@d3y419 I don't see you ever made my requests changes?

File: R/zchunk_LA112.U.R (hunk 1 above)
Comment (d3y419) [inline R/zchunk_LA112.U.R]: DONE?

**Pull request merged**